frontend/pages/createTournament.njk

```diff
@@ -5,46 +5,18 @@
 <hr style="width:80%; margin: auto;">
 <p style="text-align: center; margin-top: 2vh;">Create a Tournament and share the results online!</p>
 <div class="divForHome2" style="height: 80vh">
-    <p style="display: inline-block"> Tournament Name:</p>
-    <input type="name" class="form-control" id="tournamentName" placeholder="Name">
-    <p>Format: </p>
-    <div id="bla"> 
-        <div class="form-check">
-            <input class="form-check-input" type="radio" name="Format" id="formatCheckBox1">
-            <label class="form-check-label" for="formatCheckBox1">
-                Draw
-            </label>
+    <form action="http://localhost:8000/tournaments" method="post">
+        <p style="display: inline-block"> Tournament Name:</p>
+        <input type="name" name="name" class="form-control" id="tournament_name" placeholder="Name">
+        <div>
+            <p style="display: inline-block;"> Admin Code: </p>
```

Comment:
In your testForm.html you used the proper label elements instead of paragraphs. That's the way to go. Please do this here as well.

Reply:
Done